ui/v2.5/src/components/Scenes/SceneCard.tsx

```diff
@@ -294,7 +299,6 @@ export const SceneCard: React.FC<ISceneCardProps> = (
         draggable={props.selecting}
       >
         {maybeRenderRatingBanner()}
-        {maybeRenderSceneStudioOverlay()}
```

Comment:
Moving the studio overlay will make it impossible to overlay it in the bottom-right corner of the thumbnail (see the css snippet in the wiki). I think there is an outstanding request to swap the resolution/duration and studio positions. 

I think the better option for now would be to make either the scene thumbnail or the studio overlay into something other than a link.

I've also heard requests to move the studio link to the card rather than over the thumbnail, but I think that can be done as a normal element rather than an overlay, and we'd have some sort of option to switch showing whatever element is required.

Reply:
Resolved it by wrapping the video section in a block so it behaves the same way it used to. I've confirmed that the wiki snippet works.